### PR TITLE
Log information about follower state/apply progress

### PR DESCRIPTION
**Pull request by jsteemann**
### Scope & Purpose

Log follower progress in supervision job that organizes failover in active failover mode.

- [ ] :hankey: Bugfix
- [x] :pizza: New feature
- [ ] :fire: Performance improvement
- [ ] :hammer: Refactoring/simplification

### Checklist

- [ ] Tests
  - [ ] **Regression tests**
  - [ ] C++ **Unit tests**
  - [ ] **integration tests**
  - [ ] **resilience tests**
- [x] :book: CHANGELOG entry made
- [ ] :books: documentation written (release notes, API changes, ...)
- [ ] Backports
  - [x] Backport for 3.10: https://github.com/arangodb/arangodb/pull/17992
  - [x] Backport for 3.9: https://github.com/arangodb/arangodb/pull/17993
  - [ ] Backport for 3.8: 

#### Related Information

- [ ] Docs PR: 
- [ ] Enterprise PR:
- [ ] GitHub issue / Jira ticket:
- [ ] Design document:

### CODE REVIEW

File: arangod/Agency/ActiveFailoverJob.cpp [hunk 1]
```diff
@@ -348,8 +348,23 @@ std::string ActiveFailoverJob::findBestFollower() {
             });
   if (!ticks.empty()) {
     TRI_ASSERT(ticks.size() == 1 || ticks[0].second >= ticks[1].second);
+
+    // log information about follower states
```
Comment (MBkkt) [inline arangod/Agency/ActiveFailoverJob.cpp:352]: Can we check log level for all statement?

**Review by MBkkt — CHANGES_REQUESTED** [review]

File: arangod/Agency/ActiveFailoverJob.cpp [hunk 2]
```diff
@@ -348,8 +348,23 @@ std::string ActiveFailoverJob::findBestFollower() {
...
+    b.openArray();
+    for (auto const& it : ticks) {
+      b.openObject();
+      b.add("server", VPackValue(it.first));
+      b.add("tick", VPackValue(it.second));
+      b.close();
+    }
+    b.close();
+    LOG_TOPIC("27a94", INFO, Logger::SUPERVISION)
+        << "follower states: " << b.slice().toJson();
```
Comment (Dronplane) [inline arangod/Agency/ActiveFailoverJob.cpp:363]: NIT 
Why not just build a string "manually"?

File: arangod/Agency/ActiveFailoverJob.cpp (hunk 1 above)
Comment (MBkkt) [reply arangod/Agency/ActiveFailoverJob.cpp:352]: For an example just lamda which return builder

**Review by neunhoef — APPROVED** [review]
LGTM

**Pull request merged**